.github/ISSUE_TEMPLATE/bug_report.md

```
@@ -18,6 +17,6 @@ assignees: ''
 
 ---
 
-## 📌 작업 상세 내용
+## 📌 TODO_List
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)